daemon/cmd/daemon.go

```diff
@@ -1065,13 +1059,30 @@ func newDaemon(ctx context.Context, cleaner *daemonCleanup, params *daemonParams
 	// router IPs from the K8s resources if we weren't able to restore them
 	// from the fs. We must do this after IPAM because we must wait until the
 	// K8s resources have been synced. Part 2/2 of restoration.
-	if option.Config.EnableIPv4 {
-		d.restoreCiliumHostIPs(false, router4FromK8s)
-	}
+	ipv6, router := false, router4FromK8s
 	if option.Config.EnableIPv6 {
-		d.restoreCiliumHostIPs(true, router6FromK8s)
+		ipv6, router = true, router6FromK8s
 	}
 
+	bo := wait.Backoff{
+		Duration: 500 * time.Millisecond,
```

Comment:
I think that's a long time to wait before retrying given this is on the critical path for daemon startup, I'd shorten this.

Reply:
Good point! I am not versed with netlink hence not sure what would be a reasonable delay to retry the service? Lowering it down to 100ms. Please pipe in if you think otherwise. Thank you!